DN1/docs/demo.jl

```diff
@@ -0,0 +1,53 @@
+#' # DN1 - Naravni zlepek
+
+#' Imamo $n$ interpolacijskih točk $(x_i, f_i)$, $i=1,2,...,n$.
+#' Naravni interpolacijski kubični zlepek $S$ je funkcija, ki izpolnjuje naslednje pogoje:
+#' 1. $S(x_i) = f_i, i=1,2,\dots,n$
+#' 2. je polinom stopnje 3 ali manj na vsakem podintervalu $[x_i, x_{i+1}], i=1,2,\dots,n-1$
+#' 3. je dvakrat zvezno odvedljiva funkcija na interpolacijskem intervalu $[x_1,x_n]$
+#' 4. $S''(x_1)=S''(x_n)=0$
```

Comment:
Zakaj izberemo ta pogoj? Razložite! Kaj to pomeni za samo interpolacijo? Ali je lahko ta pogoj izpolnjen, če imamo le dve interpolacijski točki? Kaj naredimo v tem primeru?